lua/neo-tree/sources/common/commands.lua

```diff
@@ -480,6 +480,32 @@ M.copy = function(state, callback)
   local using_root_directory = get_using_root_directory(state)
   fs_actions.copy_node(node.path, nil, callback, using_root_directory)
 end
+---Copies a node relative path to clipboard.
+---@param state table The state of the source
+M.copy_path = function(state)
+  local node = state.tree:get_node()
+  if node.type == "message" then
+    return
+  end
+  local pwdpath = vim.fn.getcwd()
```

Comment:
Shouldn't this path be relative to the root of the tree? That may or may not be the same thing as what `vim.fn.getcwd()` will return. I would do it like this:
```suggestion
  local pwdpath = state.path
```